qa-tests-backend/src/test/groovy/ImageManagementTest.groovy

```diff
@@ -10,11 +10,27 @@ import org.junit.experimental.categories.Category
 import services.CVEService
 import services.ImageService
 import services.PolicyService
-import spock.lang.IgnoreIf
 import spock.lang.Unroll
 import util.Env
 
 class ImageManagementTest extends BaseSpecification {
+
+    def suppressImageCVE(String cve) {
+        if (Env.CI_JOBNAME.contains("postgres")) {
+            CVEService.suppressImageCVE(cve)
+        } else {
+            CVEService.suppressCVE(cve)
+        }
+    }
+
+    def unsuppressImageCVE(String cve) {
+        if (Env.CI_JOBNAME.contains("postgres")) {
+            CVEService.unsuppressImageCVE(cve)
+        } else {
+            CVEService.unsuppressCVE(cve)
+        }
+    }
```

Comment:
Move this into CVEService and abstract it behind `suppressImageCVE/unsuppressImageCVE` which under the hood make the correct call based on the pg flag.